e2e/tests/components/select/forms/select-forms.e2e-spec.ts

```diff
@@ -578,4 +578,62 @@ describe('Select (forms) Tests', () => {
         expect<any>(page.getNumberOfCountries(false)).toEqual(result);
 
     });
+
+    it('should not show the clear button by default in single select', async () => {
+        expect(await page.isClearButtonPresent()).toBeFalsy();
+    });
```

Comment:
Since these are end-to-end tests, the setup between tests is quite significant. This is one of the reasons that we test use cases rather than unit test style assertions. I think there are two options here:
1. Include these assertions into the existing test cases.
2. Establish Karma tests for this purpose.

@ashley-hunter what do you think?

Reply:
We could move these to unit tests if you prefer, wouldn't be difficult to do, they were just added here because the test suite already existed not for any technical limitation/reason.

Reply:
My main concern is impact on the overall build time, so if this can feasibly be done during this sprint, and if it would save execution time, then it would be worthwhile.